packages/connect/src/device/DeviceList.ts

```diff
@@ -224,7 +224,7 @@ export class DeviceList extends TypedEmitter<DeviceListEvents> implements IDevic
         const enumerateResult = await transport.enumerate({ signal });
 
         if (!enumerateResult.success) {
-            throw new Error(enumerateResult.error);
+            throw new Error(enumerateResult.message || enumerateResult.error);
```

Comment:
I see, if it is `unexpected error` then it should also have a message.

Reply:
and then this error is propagated here right https://github.com/trezor/trezor-suite/blob/fix/connect-transport-error/packages/connect/src/core/index.ts#L1268 ? and sent o user, and we belive that it is better to show details instead of unexpected-error? right?